src/app/TargetView/TargetContextSelector.tsx

```diff
@@ -49,20 +50,23 @@ export const TargetContextSelector: React.FC<TargetContextSelectorProps> = ({ cl
   const [targets, setTargets] = React.useState<Target[]>([]);
   const [selectedTarget, setSelectedTarget] = React.useState<Target>();
   const [favorites, setFavorites] = React.useState<string[]>(getFromLocalStorage('TARGET_FAVORITES', []));
+  const [searchTerm, setSearchTerm] = React.useState<string>('');
   const [isTargetOpen, setIsTargetOpen] = React.useState(false);
   const [isLoading, setLoading] = React.useState(false);
 
-  const handleSelectToggle = React.useCallback(() => setIsTargetOpen((old) => !old), [setIsTargetOpen]);
+  const onToggleClick = () => {
```

Comment:
Let's also use `React.useCallback` here?

Reply:
Ah right, nice catch. I had it as a callback initially but had done something wrong (not sure what, now) and it wasn't working, so I rewrote it like that. But it does work properly now, after rewriting it and fixing whatever I did wrong the first time :-)